src/main/java/mate/jdbc/service/CarServiceImpl.java

```diff
@@ -0,0 +1,67 @@
+package mate.jdbc.service;
+
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.dao.CarDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+
+@Service
+public class CarServiceImpl implements CarService {
+    @Inject
+    private CarDao carDao;
+
+    @Override
+    public Car create(Car car) {
+        return carDao.create(car);
+    }
+
+    @Override
+    public Car get(Long id) {
+        return carDao.get(id).isPresent() ? carDao.get(id).get() : null;
```

Comment:
If there is no such entity, it is better to throw Exception.
Also, here we refer to the database twice, it is not necessary to do this
```suggestion
        return carDao.get(id).orElseThrow(() ->
                new NoSuchElementException("Can't get car by id: " + id));
```